app/controllers/reports_controller.rb

```diff
@@ -106,6 +106,8 @@ def compute_rpm(count, total_reads)
   def compute_zscore(count, mean, stdev)
     if count && mean
       (count - mean) / stdev
+    elsif count
+      Float::MAX
```

Comment:
I think we should have a discussion with Joe on this when there's no observation on background.

Reply:
agree